sorc/ncep_post.fd/RQSTFLD.F

```diff
@@ -2085,12 +2085,13 @@ module RQSTFLD_mod
       DATA IFILV(684),AVBL(684),IQ(684),IS(684),AVBLGRB2(684)      &
      &                      /1,'SEAS25 SFC MASS CON ',159,200,     &
                              'seas pm25 sfc mass concentration  '/
-!      DATA IFILV(685),AVBL(685),IQ(685),IS(685),AVBLGRB2(685)      &
-!     &                      /1,'DU CR SFC MASS CON  ',160,001,     &
-!                             'dust cr sfc mass con              '/
+      DATA IFILV(685),AVBL(685),IQ(685),IS(685),AVBLGRB2(685)      &
+     &                      /1,'DUST10 SFC MASS CON  ',160,001,     &
+                             'dust pm10 sfc mass con              '/
       DATA IFILV(686),AVBL(686),IQ(686),IS(686),AVBLGRB2(686)      &
      &                      /1,'DUST25 SFC MASS CON ',161,001,     &
                              'dust pm25 sfc mass con            '/
+#if 0
```

Comment:
@lipan-NOAA May I ask the intention of adding the if statement?

Reply:
I kept this part unchanged since those variable are not used in GEFS-Aerosols

Reply:
Got it.